l10n/en-US/viewer.properties

```diff
@@ -99,8 +99,8 @@ print_progress_close=Cancel
 # tooltips)
 toggle_sidebar.title=Toggle Sidebar
 toggle_sidebar_label=Toggle Sidebar
-outline.title=Show Document Outline
-outline_label=Document Outline
+document_outline_label=Show Document Outline (double-click to expand/collapse all items)
```

Comment:
The label and title IDs must be swapped, i.e., this should be the title, just like before.